web/src/app/administration/components/classification-overview/classification-overview.component.ts

```diff
@@ -35,7 +36,9 @@ export class ClassificationOverviewComponent implements OnInit, OnDestroy {
 
           if (this.routerParams.id) {
             this.showDetail = true;
-            this.store.dispatch(new SelectClassification(this.routerParams.id));
+            this.store.dispatch(new SelectClassification(this.routerParams.id))
+              .pipe(takeUntil(this.destroy$))
```

Comment:
@cnguyen-de is this pipe necessary? I don't think so.

Reply:
imo it can be removed. "The observable would complete when the dispatched action's handling (and 'child' action handling) is completed. No unsubscription necessary."

Reply:
@sofie29 you can remove it then :)

Sorry for the double roundtrip

Reply:
removed